### PR TITLE
Integrate the details contact service with the front

### PR DESCRIPTION
**Pull request by CSaavedra96**
## Description
Created the logic to call a contac details from the API and map the result in the UI

## Fixes #167799313
https://www.pivotaltracker.com/story/show/167799313

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## Additional notes:
Changed the attribute baseUrl to baseURL in the axios create object.
Installed the Material Dessign icons library.

### CODE REVIEW

File: ui/web/src/services/RestServices/index.js [hunk 1]
```diff
@@ -3,17 +3,17 @@ import axios from 'axios';
 export default class RestServices{
     constructor(){
         this.http = axios.create({
-            baseUrl: process.env.VUE_APP_BASE_URL,
-            headers:{ Authorization: `Bearer ${proccess.env.VUE_APP_ACCESS_TOKEN}` }
+            baseURL: `${process.env.VUE_APP_BASE_URL}`,
```
Comment (maumaigre) [inline ui/web/src/services/RestServices/index.js]: Not necessary to use template literal string in baseURL

**Review by MiguelDiazArkus — APPROVED** [review]
looks good to me

File: ui/web/src/views/ContactDetails.vue [hunk 2]
```diff
@@ -88,52 +89,86 @@
...
+    }
+  }),
+  mounted() {
+    const restServices = new RestServices();
+
+    restServices.getContact(this.$route.params.id).then(res => {
+      const empty = "Empty";
+      const contact = res.data;
+
+      this.contact.name = `${contact.first_name} ${contact.last_name}`;
```
Comment (maumaigre) [inline ui/web/src/views/ContactDetails.vue]: Use object declaration
this.contact = {
    fax: contact.fax,
    email: contact.email
}

Don't use "Empty" as a string in the object, display "Empty" string if needed in the template directly not on the contact object

**Review by maumaigre — CHANGES_REQUESTED** [review]

File: ui/web/src/services/RestServices/index.js (hunk 1 above)
Comment (CSaavedra96) [reply ui/web/src/services/RestServices/index.js]: Fixed

File: ui/web/src/views/ContactDetails.vue (hunk 2 above)
Comment (CSaavedra96) [reply ui/web/src/views/ContactDetails.vue]: Fixed and changed

**Review by maumaigre — APPROVED** [review]

**Pull request merged**